server.go

```diff
@@ -560,6 +518,8 @@ func translateErrno(errno syscall.Errno) uint32 {
 		return ssh_FX_NO_SUCH_FILE
 	case syscall.EPERM:
 		return ssh_FX_PERMISSION_DENIED
+	case syscall.ENOSYS:
+		return ssh_FX_OP_UNSUPPORTED
 	}
 
 	return ssh_FX_FAILURE
```

Comment:
will need to go over the file diff here in person... lots of questions, probably because I haven't read the SFTP spec